bip-update-process.md

```diff
@@ -0,0 +1,708 @@
+[BIP 50](bip-0050.mediawiki)), or other information relevant to the Bitcoin community. However, any topics related to
+the Bitcoin protocol, Bitcoin’s peer-to-peer network, and Bitcoin client software may be acceptable.
+
+BIPs are intended to be the primary mechanism for proposing new protocol features, coordinating client standards, and
+documenting design decisions that have gone into implementations. BIPs may be submitted by anyone.
+
+### BIP Ownership
+
+Each BIP is primarily owned by its authors and represents the authors’ opinion or recommendation. The authors are
+expected to foster discussion, address feedback and dissenting opinions, and, if applicable, advance the adoption of
+their proposal within the Bitcoin community.
+
+#### Authors and Shepherds
+
+Authors may want additional support with the BIP process after writing an initial draft. In that case, they may assign
+one or more Shepherds to their BIP. Shepherds are stand in owners of a BIP who were not involved in writing the
+document. They support the authors in advancing the proposal, or act as point-of-contact for the BIP in absence of the
+authors. Shepherds may perform the role of Authors for any aspect of the BIP process unless overruled by an Author.
+Shepherds share ownership of the BIP at the discretion of the Authors.
+
+### What is the significance of BIPs?
+
+BIPs do not define what Bitcoin is: individual BIPs do not represent Bitcoin community consensus or a general
+recommendation for implementation. A BIP represents a personal recommendation by the BIP authors to the Bitcoin
+community. Some BIPs may never be adopted. Some BIPs may be adopted by one or more Bitcoin clients or other related
+software. Some may even end up changing the consensus rules that the Bitcoin ecosystem jointly enforces.
+
+### What is the purpose of the BIP repository?
+
+The [BIP repository](https://github.com/bitcoin/bips/) serves as a publication medium and archive for mature proposals.
+Through its high visibility, it facilitates the community-wide consideration of BIPs and provides a well-established
+source to retrieve the latest version of any BIP. The repository transparently records all changes to each BIP and
+allows any community member to retain a complete copy of the archive easily.
+
+The BIP repository is not a tool to track acceptance[^acceptance], adoption, or community consensus on BIPs, beyond
+providing a brief overview of BIP statuses (see [Workflow](#workflow) below) to the audience.
+
+### What is the scope of the BIP repository?
+
+The BIP repository is focused on information and technologies that aim to support and expand the utility of the bitcoin
+currency. Related topics that are of interest to the Bitcoin community may be acceptable. The scope of the BIP
+repository is limited to BIPs that do not oppose the fundamental principle that Bitcoin constitutes a peer-to-peer
+electronic cash system for the bitcoin currency.
```

Comment:
Line 33-42 above also appear to discuss the scope. Suggest a single point of truth in this document that defines the scope of BIPs.